wordpress-mobile/gutenberg-mobile
# Sync RichText with web counterparts

This PR updates the rich text components in GB-mobile to be more in line with their web counterparts.

All the work is done in this GB PR: https://github.com/WordPress/gutenberg/pull/17897

To test:
 Because of the nature of the changes a lot of editing experience needs to be retested, please check the following:
 - Selection of text blocks by tapping directly on the text part.
 - Writing on text blocks (paragraph, heading, lists, quotes)
 - Split and merge of text blocks (try on all text blocks and also merges of different kind of blocks)
 - Try split and merge on lists with indents
 - Test copy/paste of:
   - normal text
   - html text
   - images
Update release notes:
